cmd/ntp/ntp.go

```diff
@@ -40,6 +40,10 @@ func set(rtc *os.File) error {
 func main() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
+	if len(os.Args) > 1 {
+		servers = os.Args[1:]
```

Comment:
Just to make sure that we can add flags in the future, could you change this function to call `flag.Parse()`, then use `flag.Args()`?

Reply:
Ah. Much better.

Reply:
I don’t see a change yet? Did you forget to force-push the branch maybe?

Reply:
I did not push yet as I decided to also add a log line to show NTP servers in use.
It's coming.

Reply:
deciding the 6 words logging string took ~15 minutes of going through alternatives and not liking any. please change the wording as you see it.